addon/components/power-calendar/days.js

```diff
@@ -199,7 +209,7 @@ export default Component.extend({
     }
 
     let disabledDates = this.get('disabledDates');
-    if (disabledDates && disabledDates.some((d) => momentDate.isSame(d, 'day'))) {
+    if (disabledDates && disabledDates.some((d) => momentDate.isSame(d, 'day') || (WEEK_DAYS.includes(d) && momentDate.format('ddd') === d))) {
```

Comment:
I just realised thjat `WEEK_DAYS` is a plain JS array, not an `Ember.A`, and that means that `includes` will not work in InternetExporer. Please use `indexOf() > -1`